src/test/java/teammates/test/pageobjects/InstructorFeedbackEditPage.java

```diff
@@ -1257,11 +1258,36 @@ public void fillMcqOptionForNewQuestion(int optionIndex, String optionText) {
         fillTextBox(optionBox, optionText);
     }
 
+    public void fillMcqOption(int qnNumber, int optionIndex, String optionText) {
+        WebElement optionBox = browser.driver.findElement(By.id("mcqOption-" + optionIndex + "-" + qnNumber));
+        fillTextBox(optionBox, optionText);
+    }
+
+    public void dragAndDropMcqOption(int qnNumber, int sourceIndex, int targetIndex) throws Exception {
+        WebElement draggedOptionElement = browser.driver.findElement(By.xpath("//div[@id='mcqOptionRow-"
+                + sourceIndex + "-" + qnNumber + "']//span[@class='glyphicon glyphicon-resize-vertical']"));
+        WebElement targetElement = browser.driver.findElement(By.xpath("//div[@id='mcqOptionRow-"
+                + targetIndex + "-" + qnNumber + "']//span[@class='glyphicon glyphicon-resize-vertical']"));
+
+        Actions builder = new Actions(browser.driver);
+        builder.clickAndHold(draggedOptionElement).perform();
+        builder.moveToElement(targetElement, 0, 40).perform();
+        builder.moveToElement(targetElement, 0, 30).perform();
+        builder.moveToElement(targetElement, 0, 20).perform();
+        builder.moveToElement(targetElement).perform();
+        builder.moveToElement(targetElement, 0, -10).release().perform();
```

Comment:
> what delay are u referring to in ur comment?

By delay I meant that the dragging motion should be slow enough so that the elements shift to create empty space. Normally that doesn't happen in selenium since movements are instantaneous.

> hv u considered dragAndDropBy

yes i did come across that as well but it doesn't really help since i want to simulate a "slow dragging motion" and all these functions simulate the movement instantaneously.

Reply:
>By delay I meant that the dragging motion should be slow enough so that the elements shift to create empty space. Normally that doesn't happen in selenium since movements are instantaneous.

What about u shift instantaneous and wait for a while for the empty space? I think in terms of readability the code will be easier to understand

Reply:
Yes i did try that but the issue is not waiting before releasing. The shift itself needs to be slow/delayed. I agree the current simulation is complex so i experimented a bit and simplified it to just 3 movements as shown below:
![image](https://user-images.githubusercontent.com/11634699/41851234-1f076e02-787f-11e8-9db0-79c72cb42f50.png)
I have tested it on distribute points questions and it works fine